lib/bolt/cli.rb

```diff
@@ -146,11 +148,15 @@ def create_option_parser(results)
                 "Parameters to a task or plan") do |params|
           results[:task_options] = parse_params(params)
         end
-        results[:insecure] = false
         opts.on('-k', '--insecure',
                 "Whether to connect insecurely ") do |insecure|
           results[:insecure] = insecure
         end
+        opts.on('--transport TRANSPORT', TRANSPORTS,
+                "Specify which transport to use,
```

Comment:
Specify the default transport.

Reply:
The line wrapping was wrong too, fixed that